CHANGELOG.md

```diff
@@ -1,23 +1,21 @@
-## [CAPI Control Plane Provider Talos 0.4.0-beta.0](https://github.com/talos-systems/cluster-api-control-plane-provider-talos/releases/tag/v0.4.0-beta.0) (2021-11-29)
+## [CAPI Control Plane Provider Talos 0.4.0-beta.1](https://github.com/talos-systems/cluster-api-control-plane-provider-talos/releases/tag/v0.4.0-beta.1) (2021-12-09)
 
-Welcome to the v0.4.0-beta.0 release of CAPI Control Plane Provider Talos!  
+Welcome to the v0.4.0-beta.1 release of CAPI Control Plane Provider Talos!  
```

Comment:
We had beta.0 release, right? So we should be merging changelogs, not replacing them?